Royalty/DefInjected/QuestScriptDef/Script_Hospitality_Worker.xml

```diff
@@ -2,67 +2,67 @@
-  <Hospitality_Util_Worker.LetterTextLodgersArrived.value.slateRef>TODO</Hospitality_Util_Worker.LetterTextLodgersArrived.value.slateRef>
+  <Hospitality_Util_Worker.LetterTextLodgersArrived.value.slateRef>[lodgersLabelSingOrPluralDef] [lodgersIsAre] aangekomen. Bescherm [lodgersObjective] totdat de shuttle arriveert over [shuttleDelayTicks_duration].[helpersArrivalLetterEnd]</Hospitality_Util_Worker.LetterTextLodgersArrived.value.slateRef>
   <!-- EN: Should depart on shuttle -->
-  <Hospitality_Util_Worker.InspectStringDepartShuttle.slateRef>TODO</Hospitality_Util_Worker.InspectStringDepartShuttle.slateRef>
+  <Hospitality_Util_Worker.InspectStringDepartShuttle.slateRef>Moet met de shuttle vertrekken</Hospitality_Util_Worker.InspectStringDepartShuttle.slateRef>
   <!-- EN: Shuttle arrived -->
-  <Hospitality_Util_Worker.LetterLabelShuttleArrived.slateRef>TODO</Hospitality_Util_Worker.LetterLabelShuttleArrived.slateRef>
+  <Hospitality_Util_Worker.LetterLabelShuttleArrived.slateRef>Shuttle gearriveerd</Hospitality_Util_Worker.LetterLabelShuttleArrived.slateRef>
   <!-- EN: The shuttle has arrived to collect [lodgersLabelSingOrPluralDef]. Be sure to have these people on the shuttle before it leaves in [shuttleLeaveDelayTicks_duration]:\n\n[lodgers] -->
-  <Hospitality_Util_Worker.LetterTextShuttleArrived.slateRef>TODO</Hospitality_Util_Worker.LetterTextShuttleArrived.slateRef>
+  <Hospitality_Util_Worker.LetterTextShuttleArrived.slateRef>De shuttle is aangekomen om [lodgersLabelSingOrPluralDef] op te halen. Zorg ervoor dat je deze mensen in de shuttle hebt voordat deze vertrekt over [shuttleLeaveDelayTicks_duration]:\n\n[lodgers]</Hospitality_Util_Worker.LetterTextShuttleArrived.slateRef>
   <!-- EN: Guest died: {SUBJECT_definite} -->
-  <Hospitality_Util_Worker.LetterLabelGuestDied.slateRef>TODO</Hospitality_Util_Worker.LetterLabelGuestDied.slateRef>
+  <Hospitality_Util_Worker.LetterLabelGuestDied.slateRef>Gast overleden: {SUBJECT_definite}</Hospitality_Util_Worker.LetterLabelGuestDied.slateRef>
   <!-- EN: [letterSubject], who you were charged to protect, has died. [remainingWillNowLeave][failLetterEndingCommon] -->
-  <Hospitality_Util_Worker.LetterTextGuestDied.slateRef>TODO</Hospitality_Util_Worker.LetterTextGuestDied.slateRef>
+  <Hospitality_Util_Worker.LetterTextGuestDied.slateRef>[letterSubject], die je moest beschermen, is overleden. [remainingWillNowLeave][failLetterEndingCommon]</Hospitality_Util_Worker.LetterTextGuestDied.slateRef>
   <!-- EN: Guest captured: {SUBJECT_definite} -->
-  <Hospitality_Util_Worker.LetterLabelGuestCapture.slateRef>TODO</Hospitality_Util_Worker.LetterLabelGuestCapture.slateRef>
+  <Hospitality_Util_Worker.LetterLabelGuestCapture.slateRef>Gast gevangen genomen: {SUBJECT_definite}</Hospitality_Util_Worker.LetterLabelGuestCapture.slateRef>
```

Comment:
Ik moest het even dubbelchecken in de vandale, maar gevangengenomen is één woord :) 
https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/gevangennemen

Alternatief, gearresteerd? Dat wordt op andere plaatsten ook gebruikt zie ik.